doc/eng_sys_checks.md

```diff
@@ -32,7 +32,10 @@
 
 There are various tests currently enabled in Azure pipeline for Python SDK and some of them are enabled only for nightly CI checks. We also run some static analysis tool to verify code completeness, security and lint check.
 
-Check the [contributing guide](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md#building-and-testing) for an intro to `tox`.
+Check the [contributing guide](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md#building-and-testing) for an intro to `tox`. For a deeper dive into the tooling that enables the CI checks below and additional detail on reproducing builds locally please refer to the azure-sdk-tools README.md.
+
+TODO: publish github.io
+The development documentation for this repository is published at []().
```

Comment:
Is this something that can be done now, or needs to be done after getting this merged?

Reply:
I will clean up todos, but this piece will get done after merged. I will merge this today after resolving the regression failures.